### PR TITLE
Replacing all images with Imgur.

### PR DESCRIPTION
**Pull request by snowy1000**

### CODE REVIEW

File: websites/D/Dizipub/dist/metadata.json [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
+      "name": "Snowy ✨",
+      "id": "693445343332794408"
+    },
+    "service": "Dizipub",
+    "description": {
+      "en": "Dizipub is a website that you can watch your favorite series and films.",
+      "tr": "Dizipub, favori film ve dizilerinizi kolayca bulabileceğiniz bir film sitesidir."
+    },
+    "url": "dizipub.net",
+    "version": "1.0.0",
```
Comment (Bas950) [inline websites/D/Dizipub/dist/metadata.json:12]: v 1.0.2

File: websites/W/WoxeBot/dist/metadata.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+      "name": "Snowy ✨",
+      "id": "693445343332794408"
+    },
+    "service": "WoxeBot",
+    "description": {
+      "en": "WoxeBot is a Discord bot that you can add to your server to manage it easily with confidence!",
+      "tr": "WoxeBot sunucularınıza güvenle ekleyebileceğiniz ve işinizi kolaylaştıracak bir Discord botudur"
+    },
+    "url": "panel.woxe.me",
+    "version": "1.0.0",
```
Comment (Bas950) [inline websites/W/WoxeBot/dist/metadata.json:12]: same here

File: websites/R/Ralex Bot/dist/metadata.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
   },
   "url": "bot.ralex.xyz",
   "version": "1.0.1",
```
Comment (Bas950) [inline websites/R/Ralex Bot/dist/metadata.json:12]: same here

File: websites/D/Dizipub/dist/metadata.json (hunk 1 above)
Comment (snowy1000) [reply websites/D/Dizipub/dist/metadata.json]: Updated.

File: websites/W/WoxeBot/dist/metadata.json (hunk 2 above)
Comment (snowy1000) [reply websites/W/WoxeBot/dist/metadata.json]: Updated.

File: websites/R/Ralex Bot/dist/metadata.json (hunk 3 above)
Comment (snowy1000) [reply websites/R/Ralex Bot/dist/metadata.json]: Updated.

**Pull request merged**